### ISSUE TITLE
Maintenance spot-checks for fractions string reference change

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/QA#463]
For https://github.com/phetsims/rosetta/issues/213, it's desired to have the fractions sims switch to use the common string (that's used in master) since rosetta is checking things in master. fraction-matcher and fractions-equality were published recently enough where this change was applied, but the three other sims in the suite needed changes.

I'd like QA to do brief checks to make sure nothing horrible broke (make sure the strings are working, and if translations are in the RC directories to test a few), and @jbphet if you can ensure that these look good using the vegas string.

@ariel-phet, let me know if for some reason I shouldn't production-release this once confirmed (due to increasing the app download)

build-a-fraction 1.0 (https://github.com/phetsims/rosetta/issues/213)

- [ ] [build-a-fraction 1.0.11-rc.1](https://phet-dev.colorado.edu/html/build-a-fraction/1.0.11-rc.1/phet/build-a-fraction_en_phet.html)

fractions-intro 1.0 (https://github.com/phetsims/rosetta/issues/213)

- [ ] [fractions-intro 1.0.11-rc.1](https://phet-dev.colorado.edu/html/fractions-intro/1.0.11-rc.1/phet/fractions-intro_en_phet.html)

fractions-mixed-numbers 1.0 (https://github.com/phetsims/rosetta/issues/213)

- [ ] [fractions-mixed-numbers 1.0.11-rc.1](https://phet-dev.colorado.edu/html/fractions-mixed-numbers/1.0.11-rc.1/phet/fractions-mixed-numbers_en_phet.html)

**Comment by ariel-phet** [comment]
@jonathanolson that is fine for app download, we are generally OK with small updates with that, just want to be judicious when it comes to releases affecting every sim.

**Comment by jbphet** [comment]
>  @jbphet if you can ensure that these look good using the vegas string...

I verified this by adding special-case code to Rosetta (running locally on my machine, never committed) that substituted the RC links above for the live sim URLs from which the English string keys are usually extracted.  This worked, i.e. the "Choose your Level!" string was presented on the translation utility interface, here's a screenshot:

![image](https://user-images.githubusercontent.com/3082831/71219364-1284b000-2282-11ea-9a17-df66000be81c.png)

I also tested this using the "Test" function, and it looked good too.  Below is a screenshot - note that since I don't speak Persian, I just pasted the string for "credits" into the "chooseYourLevel" string.

![image](https://user-images.githubusercontent.com/3082831/71219202-81153e00-2281-11ea-9e48-1c1aa40c8a3e.png)

So, from the translation perspective, these changes are good to go.
